rpc-rs/src/rpc_client.rs

```diff
@@ -244,7 +246,7 @@ impl RpcClient {
         let subject = make_uuid();
         let issuer = &self.key.public_key();
         let target_url = format!("{}/{}", target.url(), &message.method);
-        debug!("rpc_client sending to {}", &target_url);
+        debug!(%target_url, "rpc_client sending");
```

Comment:
this debug would be more useful if we split up the url into fields public_key, operation, link, and contract, since that will match the trace variable names used elsewhere.

Reply:
Let me know if I addressed this properly